recipes/jsonextended/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "jsonextended" %}
+{% set version = "0.7.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 251364a566a8753c699caec696bcd0bfdfd782dc327d3292b254330670f56bfb
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - ijson
+    - pathlib2
+    - pint
+    - pip
+    - python
```

Comment:
you probably need only `python` and `pip` in the `host` section.

Reply:
Ah yes I thought that might be the case (`conda skeleton` put them there). Now fixed

Reply:
Actually, because the top-level of the package is imported in setup.py (to get the `__version__`), at least pathlib2 is required in host